commands/cli/parse.go

```diff
@@ -12,9 +11,12 @@ import (
 
 	cmds "github.com/ipfs/go-ipfs/commands"
 	files "github.com/ipfs/go-ipfs/commands/files"
+	logging "gx/ipfs/QmYtB7Qge8cJpXc4irsEp8zRqfnZMBeB7aTrMEkPk67DRv/go-log"
```

Comment:
This is wrong go-log version. You either used old version or need to rebase on top of master.